### PR TITLE
More styling bug fixes

### PR DESCRIPTION
**Pull request by brandonhundt**
- [ ] fix spacing issues with discard button
- [ ] Adds required field styling options (see podcast settings metadata for example on title)

### CODE REVIEW

File: app/assets/stylesheets/shared/form.scss [hunk 1]
```diff
@@ -62,6 +62,18 @@
   }
 }
 
+label[required="required"],
+label[required="true"] {
```
Comment (brandonhundt) [inline app/assets/stylesheets/shared/form.scss:66]: One of these can likely be removed after you do your thing noting required labels.

File: app/javascript/controllers/unsaved_controller.js [hunk 2]
```diff
@@ -95,9 +95,9 @@ export default class extends Controller {
 
     if (this.hasDiscardTarget) {
       if (changed) {
-        this.discardTarget.classList.remove("invisible")
+        this.discardTarget.classList.remove("d-none")
       } else {
-        this.discardTarget.classList.add("invisible")
+        this.discardTarget.classList.add("d-none")
```
Comment (brandonhundt) [inline app/javascript/controllers/unsaved_controller.js:100]: This allows for bootstrap to handle the display.

File: app/views/episodes/_form_status.html.erb [hunk 3]
```diff
@@ -41,9 +41,8 @@
 
       <% if policy(episode).update? %>
         <div>
-          <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-discard-changed invisible", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
-
           <div class="btn-group">
+            <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-discard-changed d-none me-2", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
```
Comment (brandonhundt) [inline app/views/episodes/_form_status.html.erb:45]: If the discard button is within the "btn-group" it helps a lot of the default spacing, ensures the buttons remain inline.

File: app/views/podcasts/_form_main.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <div class="col-12 mb-4">
   <div class="form-floating input-group">
     <%= form.text_field :title %>
-    <%= form.label :title %>
+    <%= form.label :title, required: "true" %>
```
Comment (brandonhundt) [inline app/views/podcasts/_form_main.html.erb:4]: This is likely temporary until. I'm assuming theres a way to get this into the model.

File: app/views/episodes/_form_status.html.erb (hunk 3 above)
Comment (cavis) [reply app/views/episodes/_form_status.html.erb:45]: This does ruin the rounded corners on the left side of the save button?

File: app/assets/stylesheets/shared/form.scss (hunk 1 above)
Comment (cavis) [reply app/assets/stylesheets/shared/form.scss:66]: I could see this being even larger/bolder!  But- that seems tricky to align.  So this is a start!

File: app/views/podcasts/_form_main.html.erb (hunk 4 above)
Comment (cavis) [reply app/views/podcasts/_form_main.html.erb:4]: I'm probably okay with them being in the views.  But we'll see.

**Review by cavis — APPROVED** [review]
👍 One issue with sharp corners.  But I kind of want to merge this, so I'll let you sort that later.

**Pull request merged**